### ISSUE TITLE
Bad status line error when connecting to radarr and sonarr

### ISSUE THREAD

**Issue by icebal** [morpheus65535/bazarr#132]
Below is the error i get when connecting ssl'd sonarr and radarr, which are reverse proxied by traefik, as is bazarr. Openssl s_client test shows the certs are coming in fine, and i am able to send api calls with curl To sonarr inside the bazarr container. 
Quick google search says sending useragent headers could be a fix?

01/09/2018 18:42:32|ERROR|Error trying to get series from Sonarr. Connection Error.|Traceback (most recent call last):  File "/bazarr/get_series.py", line 31, in update_series    r = requests.get(url_sonarr_api_series, timeout=15)  File "/usr/lib/python2.7/site-packages/requests/api.py", line 72, in get    return request('get', url, params=params, **kwargs)  File "/usr/lib/python2.7/site-packages/requests/api.py", line 58, in request    return session.request(method=method, url=url, **kwargs)  File "/usr/lib/python2.7/site-packages/requests/sessions.py", line 512, in request    resp = self.send(prep, **send_kwargs)  File "/usr/lib/python2.7/site-packages/requests/sessions.py", line 622, in send    r = adapter.send(request, **kwargs)  File "/usr/lib/python2.7/site-packages/requests/adapters.py", line 495, in send    raise ConnectionError(err, request=request)ConnectionError: ('Connection aborted.', BadStatusLine('\x15\x03\x01\x00\x02\x02\n',))|

**Comment by morpheus65535** [comment]
Why don’t you try connecting directly to Sonarr and Radarr without going trough your reverse proxy?

**Comment by icebal** [comment]
they are docker containers that are only exposed internally to the docker network, like how kubernetes pods are, with traefik as the ingress. will try for TS real quick

**Comment by icebal** [comment]
quick update, it seems that \x15\x03\x01\x00\x02\x02 is the beginning header for https connections, and this error is from expecting http but recieving https response back. i have ssl enabled for both radarr and sonarr, and get the same response with ssl option disabled in settings. adding my cert to sonarr to test with without traefik reverse proxying, and i get the same thing.

tl;dr, it seems an issue with https connections between bazarr and sonarr

**Comment by morpheus65535** [comment]
Ok thanks, I'll look into this. I'm glad you gave me all those information, it's going to help me a lot.

**Comment by morpheus65535** [comment]
Could you test the latest commit to master branch? Should be working now.

**Comment by icebal** [comment]
It works!!!!! Glad it was a an easier fix lol.

**Comment by morpheus65535** [comment]
Yes, thank you for reporting this!
